pages/mobile/details.py

```diff
@@ -68,10 +69,11 @@ def click_view_reviews(self):
         from pages.mobile.reviews import Reviews
         return Reviews(self.testsetup)
 
-    def login_with_user_from_other_pages(self, user="default"):
-        from pages.fxa import FirefoxAccounts
-        fxa = FirefoxAccounts(self.testsetup)
-        fxa.login_user(user)
+    def login_with_user_from_other_pages(self):
+        credentials = FxATestAccount(use_prod=False).create_account()
```

Comment:
This is confusing too. The method is called `login_with_user_from_other_pages`, and this one *always* creates a new user account, whereas `login` has the option of passing in an existing user.